src/Middleware/Consume/MiddlewareFactoryConsume.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Queue\Middleware\Consume;
```

Comment:
What does "consume" stand for here? Do you mean "consumer"?

Reply:
Consume-related middleware. The same as "push" in `Yiisoft\Yii\Queue\Middleware\Push`.